lib/modules/platform/github/index.md

```diff
@@ -1,5 +1,69 @@
 # GitHub and GitHub Enterprise
 
+## Authentication
+
+First, [create a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for the bot account, select `repo` scope.
+
+Let Renovate use your PAT by doing _one_ of the following:
+
+- Set your PAT as a `token` in your `config.js` file
+- Set your PAT as an environment variable `RENOVATE_TOKEN`
+- Set your PAT when you run Renovate in the CLI with `--token=`
+
+Remember to set `platform=github` somewhere in your Renovate config file.
+
+If you use GitHub Enterprise Server then `endpoint` must point to `https://github.enterprise.com/api/v3/`.
+You can choose where you want to set `endpoint`:
+
+- In your `config.js` file
+- In a environment variable
+- In a CLI parameter
+
+## Running as a GitHub App
+
+Instead of a bot account with a Personal Access Token you can run `renovate` as a self-hosted [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps).
+
+When creating the GitHub App give it the following permissions:
+
+- Checks: Read & write
+- Contents: Read & write
+- Issues: Read & write
+- Metadata: Read-only
+- Pull requests: Read & write
+- Commit statuses: Read & write
+- Dependabot alerts: Read-only
+- Workflows: Read & write
+- Members: Read
```

Comment:
This might work better as a table?

| Permission | Scope | 
| --- | --- |
| Checks | `read` and `write` |
| Contents | `read` and `write` |